packages/web-react/src/components/Toast/README.md

```diff
@@ -263,13 +264,108 @@ const [isOpen, setIsOpen] = React.useState(false)
+| `close`     | `function`                                                   | () => {}   | Function that will close UncontrolledToast               |
+| `color`     | [[Emotion Color dictionary][dictionary-color] \| `inverted`] | `inverted` | Color variant                                            |
+| `iconName`  | `string`                                                     | —          | Name of a custom icon to be shown along the message      |
+| `id`        | `string`                                                     | `''`       | ToastBar ID                                              |
+| `isOpen`    | `bool`                                                       | `false`    | Open state of UncontrolledToast                          |
+| `message`   | [`string` \| `ReactNode`]                                    | null       | Message inside UncontrolledToast                         |
+| `showToast` | `function`                                                   | () => {}   | Function that will open UncontrolledToast                |
+
+#### How to use `showToast` function:
+
+This function has two required parameters: message and ID.
+All other options are not required and can be omitted entirely.
+
+```jsx
+              ┌─⫸ Message inside UncontrolledToast (required)
+              │
+              │                 ┌─⫸ ToastBar ID (required)
+              │                 │
+showToast('Toast message', 'toast-id', {
+  autoClose: true         // Automatically close toast after 5000ms, default: false
+  color: 'danger'         // Color variant, default: 'inverted'
+  iconName: 'download'    // Name of a custom icon to be shown along the message, default: undefined
+})
+```
+
+### API
+
+| Name            | Type                                                        | Default    | Required | Description                                 |
+| --------------- | ----------------------------------------------------------- | ---------- | -------- | ------------------------------------------- |
+| `alignmentX`    | [[AlignmentX dictionary][dictionary-alignment] \| `object`] | `center`   | ✕        | Horizontal alignment of the toast queue     |
+| `alignmentY`    | [`top` \| `bottom` \| `object`]                             | `bottom`   | ✕        | Vertical alignment of the toast queue       |
+| `closeLabel`    | `string`                                                    | `Close`    | ✕        | Close label                                 |
+| `hasIcon`       | `bool`                                                      | `false` \* | ✕        | If true, an icon is shown along the message |
+| `isDismissible` | `bool`                                                      | `false`    | ✕        | If true, ToastBar can be dismissed by user  |
+
+(\*) For each emotion color, a default icon is defined.
```

Comment:
am I not able to change this icon?

Reply:
you can change it with call function `showToast` from `useUncontrolledToast` same as `color` and `autoClose`

<img width="854" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/26870554/60045230-8760-4bbb-88d1-ba8bfac56eaf">